src/components/team-bio-card/team-bio-card.tsx

```diff
@@ -18,23 +19,13 @@ interface TeamBioCardProps {
 export default function TeamBioCard({ data }: TeamBioCardProps) {
   return (
     <Box className="member-card">
-      {data.profile_image_link ? (
-        <Image
-          className="member-card__image"
-          alt="profile"
-          src={data.profile_image_link}
-          width={202}
-          height={206}
-        />
-      ) : (
-        <Image
-          className="member-card__image"
-          alt="profile"
-          src="/assets/profile-image-placeholder.png"
-          width={202}
-          height={206}
-        />
-      )}
+      <Image
+        className="member-card__image"
+        alt="profile"
+        src={data.profile_image_link}
+        width={274}
+        height={250}
```

Comment:
![image](https://github.com/CodeForBc/codeforbc-web/assets/16601729/86b77a17-5d2d-470e-ab7e-c6ec992cac60)

`NextJS` appears to be raising warnings, because we're already changing these values in the CSS, but setting these to `0` seems to make the warnings go away

```suggestion
        width={0}
        height={0}
```

![image](https://github.com/CodeForBc/codeforbc-web/assets/16601729/c7b1f7ee-eb72-465d-89d6-739658ee41e9)